PipelinesTasks/PRMetrics/buildAndReleaseTask/Updaters/CodeMetrics.ts

```diff
@@ -0,0 +1,429 @@
+/* eslint-disable spaced-comment */
```

Comment:
Please remove these. They're not required as we've got an eslint file containing the definitons alreayd